tests/unit/api/test_energy.py

```diff
@@ -15,13 +16,19 @@ def test_instantiate_energy():
 
 def test_calculation_electricity():
     """Test whether electricity emissions are calculated correctly"""
-    energy = Energy(consumption=300).from_electricity(country_code="DE")
-    assert isinstance(energy, tuple)
-    assert isinstance(energy[0], float)
+    energy = (
+        Energy(consumption=300).from_electricity(country_code="DE").calculate_co2e()
+    )
+    assert isinstance(energy, Emissions)
+    assert isinstance(energy.co2e, float)
 
 
 def test_calculation_heating():
     """Test whether heating emissions are calculated correctly"""
-    energy = Energy(consumption=300, fuel_type="gas").from_heating(unit="m^3")
-    assert isinstance(energy, tuple)
-    assert isinstance(energy[0], float)
+    energy = (
+        Energy(consumption=300, fuel_type="gas")
```

Comment:
Since `unit` is a parameter in `from_heating()` now, wouln't it make sense to put the parameter `fuel_type` there as well instead of in `Energy()`?

Reply:
We thought we'd move only the parameters to `from_heating()` and `from_electricity()` that are unique/specific to those two energy cases.. and since they share `fuel_type`, we left it there. To me, this seems to be analogous to how we did it in the transport case, do you have a different view on that?

Reply:
ah ok, yes then it's fine 👍